podcast/the-changelog-332.md

```diff
@@ -316,9 +316,9 @@ It's not just me working on the project; I think we ran the numbers recently and
 
 **Rich Harris:** Absolutely. People talk about it like a well-kept secret. This isn't my first big open source project, I've been involved in a few different ones now, and the thing that I've learned is that it takes so much longer than you would imagine to gain the level of traction that you're talking about. People still talk about Vue as like the new shiny thing. It's been around since like 2013, it's six years old; there's nothing new and shiny about a lot of the projects that are only now reaching people's awareness... So Svelte is very much under the radar for the vast majority of Javascript programmers.
 
-\[00:56:05.14\] People are starting to \[unintelligible 00:56:04.08\] during the State of Javascript 2018 Survey at the end of last year, and for the front-end framework section, Svelte was the number one write-in answer to the question "Which framework are you using?" It was ahead of Aurelia and other frameworks like that. So it's slowly gaining traction.
+\[00:56:05.14\] People are starting to \[unintelligible 00:56:04.08\] to it; there was the State of Javascript 2018 Survey at the end of last year, and for the front-end framework section, Svelte was the number one write-in answer to the question "Which framework are you using?" It was ahead of Aurelia and other frameworks like that. So it's slowly gaining traction.
 
-A little bit of anecdata - I've noticed a sudden uptick in invitations to speak at conferences, and things like that, so I think people are kind of aware that this is a thing, but a lot of people are hesitant to dip their toes into a new technology stack because there is a very high switching cost. I guess our task over the coming months is to try and persuade people that the cost of not switching is ultimately higher in the long-term than the cost of -- I confused myself...
+A little bit of anecdote - I've noticed a sudden uptake in invitations to speak at conferences, and things like that, so I think people are kind of aware that this is a thing, but a lot of people are hesitant to dip their toes into a new technology stack because there is a very high switching cost. I guess our task over the coming months is to try and persuade people that the cost of not switching is ultimately higher in the long-term than the cost of -- I confused myself...
```

Comment:
Both of these look correct in the previous version -- pretty sure Rich says uptick, and he definitely says [anecdata](https://www.dictionary.com/browse/anecdata).

Reply:
My bad! I didn't know that word existed, but you're right! 👍